_posts/2024-02-17-openttd-truetype-fonts.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TrueType fonts for OpenTTD
+author: Zephyris
+---
+
+How do you update a 30-year old pixel font to make it more suitable for today?
+
+The pixel-art OpenTTD fonts are iconic, but were designed for 640x480 CRT monitors rather than today's 4k monsters.
+Without the horizontal smoothing of CRT scanlines, modern pixel-perfect rendering and 2x or 4x upscaling of the old bitmap/sprite-based fonts makes the text jagged and hard to read.
+
+<!-- more -->
+
+![Composite of a simulated CRT and modern screenshot of OpenTTD]({% link /static/img/post_2024-02-06-openttd-truetype-fonts/crt.png %}){:width="100%"}
```

Comment:
Shouldn't the folder for the images should also get the date update?

Reply:
Hard to do, I'm on remote desktop from my phone! I don't think it was for the signals post, but would be tidier

Reply:
Well, I went for it anyway! I'd appreciate a double check for potential broken image links though.